### ISSUE TITLE
The compiling failed with openblas 

### ISSUE THREAD

**Issue by qingqing01** [PaddlePaddle/Paddle#5522]
Error:

```
[ 98%] Built target softmax_with_cross_entropy_op
/home/xxx/.third_party/install/openblas/lib/libopenblas.a(memory.o): In function `openblas_fork_handler':
memory.c:(.text+0x190): undefined reference to `pthread_atfork'
```

From this link https://github.com/xianyi/OpenBLAS/wiki/faq :

It seems that needs to link the pthread  library:

> How to solve undefined reference errors when statically linking against libopenblas.a

> On Linux, if OpenBLAS was compiled with threading support (USE_THREAD=1 by default), custom programs statically linked against libopenblas.a should also link to the pthread library e.g.:

>  gcc -static -I/opt/OpenBLAS/include -L/opt/OpenBLAS/lib -o my_program my_program.c -lopenblas -lpthread

**Comment by tensor-tang** [comment]
我也遇到这个问题

**Comment by luotao1** [comment]
@helinwang Do we need add openblas (`WITH_MKLDNN=OFF WITH_MKLML=OFF WITH_GPU=OFF` ) in our teamcity? Since this issue and #5523 are both compiled fail with openblas version.

Current CPU version in teamcity is `WITH_MKLDNN=ON WITH_MKLML=ON`.

**Comment by tensor-tang** [comment]
@helinwang that is the issue #5523 , and I gave a PR #5521 , you can help review it.

**Comment by NHZlX** [comment]
Is this error fixed?  I meet the same problem..

**Comment by luotao1** [comment]
I am working on it, and don't fix it sucessfully.
The reason caused is https://github.com/PaddlePaddle/Paddle/commit/0ede2a731120966dc0171b55eb403b2ec90f8fd8 
If I delete `USE_NO_KERNEL_OP(fill_constant)`, I can build successfully. But the unit test fails.
@reyoung can you help to see it as well ?

**Comment by luotao1** [comment]
@helinwang This issue is fixed by #5552 , you could add openblas version in the teamcity. Thanks very much!

**Comment by helinwang** [comment]
@luotao1 thanks! done.
